holmes/plugins/toolsets/infrainsights/base_toolset_v2.py

```
@@ -0,0 +1,303 @@
+        raise Exception(error_msg)
+    
+    def _generate_resolution_error_message(self, service_type: str, params: Dict[str, Any]) -> str:
+        """Generate a helpful error message when instance resolution fails"""
+        client = self.get_infrainsights_client()
+        config = client.config
+        
+        # Get available instances for reference
+        try:
+            summary = client.get_service_instance_summary(service_type)
+            available_names = summary.get('instance_names', [])
+        except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)